naccs_bot.py

```diff
@@ -159,69 +160,73 @@ def get_ongoing_matches(channel_id):
 #
 #   Get ongoing FACEIT streams for both divisions
 #
+# Loop function, seperate from main code so that we can execute both requests at the same time.
+@loop(minutes=5)
 async def get_streams():
+    await run_get_streams()
+
+async def run_get_streams():
     # Getting our globals
-  global DISPLAYEDSTREAMS
-  global DISPLAYEDSTREAMS2
-  global toggle
-  # Resetting activestreams for each division to find so we can check to make sure they are still live
-  activestreams = {}
-  activestreams2 = {}
-  # Simple toggle / workaround. Works to reduce performance inhibitance and restricts the function to one api at a time
-  if (toggle == 1):
+    global displayed_streams
+    global displayed_streams2
+    global toggle
+
+    activestreams = {}
+    activestreams2 = {}
+  
+    if (toggle == 1):
```

Comment:
Let's get rid of this 'toggle' and hit both api's during one call.

Reply:
removed in latest comit